modules/integration/tests/src/test/java/org/wso2/appmanager/integration/utils/HttpUtil.java

```diff
@@ -272,57 +130,86 @@ public static HttpResponse doPost(URL endpoint, String postBody,
      * @return httpResponse HttpResponse.
      * @throws IOException on errors.
      */
-    public static HttpResponse doGet(String endpoint,
-                                     Map<String, String> headers) throws IOException {
+    public static HttpResponse doGet(String endpoint, Map<String, String> headers) throws IOException {
+        HttpURLConnection conn = null;
```

Comment:
Fixed in https://github.com/wso2/product-app-manager/pull/357/commits/d5adfb28b2122712d0da0911c0d4921cf5bd98d9